src/analysis/retail/monk/mistweaver/modules/dragonflight/tier/T31MWTier.tsx

```diff
@@ -27,7 +27,7 @@ const TARGET_SPELLS = [
   TALENTS_MONK.SOOTHING_MIST_TALENT,
 ];
 
-const TOOLTIP_SPELLS = [...TARGET_SPELLS, SPELLS.GUSTS_OF_MISTS];
+const TOOLTIP_SPELLS = [...TARGET_SPELLS, SPELLS.GUSTS_OF_MISTS, SPELLS.GUST_OF_MISTS_CHIJI];
```

Comment:
The intention with the tooltip spells here is to specifically show your effectiveness on casting onto chi harmony targets so I'd like to avoid including non-targeted abilities